### PR TITLE
New package: e2guardian-5.3.2

### PR DESCRIPTION
**Pull request by phinas**

### CODE REVIEW

File: srcpkgs/e2guardian/files/e2guardian/run [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+E2GUARDIAN_PID=/var/run/e2guardian.pid
```
Comment (xtraeme) [inline srcpkgs/e2guardian/files/e2guardian/run]: Why is the pid file necessary?

File: srcpkgs/e2guardian/template [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Template file for 'e2guardian'
+pkgname=e2guardian
+version=5.3.2
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --enable-clamd=yes --with-proxyuser=_e2guardian --with-proxygroup=_e2guardian --sysconfdir=/etc --localstatedir=/var --enable-icap=yes --enable-commandline=yes --enable-email=yes --enable-ntlm=yes --mandir=${prefix}/share/man --infodir=${prefix}/share/info --enable-pcre=yes --enable-sslmitm=yes"
+conf_files="/etc/e2guardian/*.conf
+/etc/e2guardian/site.story
+/etc/e2guardian/preauth.story
+/etc/e2guardian/site.story
```
Comment (xtraeme) [inline srcpkgs/e2guardian/template]: Change those 3 lines from above to `/etc/e2guardian/*.story` ?

File: srcpkgs/e2guardian/template [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+distfiles="https://github.com/e2guardian/e2guardian/archive/v${version}.tar.gz"
+checksum=ff15090157b8ea7b7910a893bb3bf582b5658cd2b4a9417805bd5c8cc1ba5e29
+system_accounts="_e2guardian"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*)
+		configure_args+="-mno-sse2"
+		;;
+	i686*)
+		configure_args+="-mno-sse2"
```
Comment (xtraeme) [inline srcpkgs/e2guardian/template]: why is this necessary?

File: srcpkgs/e2guardian/template [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Template file for 'e2guardian'
+pkgname=e2guardian
+version=5.3.2
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --enable-clamd=yes --with-proxyuser=_e2guardian --with-proxygroup=_e2guardian --sysconfdir=/etc --localstatedir=/var --enable-icap=yes --enable-commandline=yes --enable-email=yes --enable-ntlm=yes --mandir=${prefix}/share/man --infodir=${prefix}/share/info --enable-pcre=yes --enable-sslmitm=yes"
```
Comment (xtraeme) [inline srcpkgs/e2guardian/template]: `--prefix` `--infodir` and `--mandir` are set by `build-style=gnu-configure`.

File: srcpkgs/e2guardian/template [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+/etc/e2guardian/lists/bannedrooms/*
+/etc/e2guardian/lists/contentscanners/*
+/etc/e2guardian/lists/phraselists/*/*
+/etc/e2guardian/authplugins/*
+/etc/e2guardian/contentscanners/*
+/etc/e2guardian/downloadmanagers/default.conf"
+make_dirs="/var/log/e2guardian 0755 _e2guardian _e2guardian"
+hostmakedepends="automake pkg-config"
+makedepends="zlib-devel pcre-devel libressl-devel"
+short_desc="E2guardian is a web content filter proxy with multiple server modes"
```
Comment (xtraeme) [inline srcpkgs/e2guardian/template]: Fix the xlint errors.

File: srcpkgs/e2guardian/files/e2guardian/run (hunk 1 above)
Comment (phinas) [reply srcpkgs/e2guardian/files/e2guardian/run]: It's actually not. I just fixed it

File: srcpkgs/e2guardian/template (hunk 3 above)
Comment (phinas) [reply srcpkgs/e2guardian/template]: I took the configure_args from the installation instructions of the project. I'm not sure why it would be necessary, so I could remove it.

File: srcpkgs/e2guardian/template (hunk 5 above)
Comment (phinas) [reply srcpkgs/e2guardian/template]: I ran xlint on the template, but it didn't throw an error for me. Could you please point out what is wrong with that part?

File: srcpkgs/e2guardian/template (hunk 3 above)
Comment (phinas) [reply srcpkgs/e2guardian/template]: I just checked again, they have this flag only in the debian section of the INSTALL instructions, not for any other distribution. I just removed it

File: srcpkgs/e2guardian/template (hunk 5 above)
Comment (not-chicken) [reply srcpkgs/e2guardian/template]: Please remove the package name from description, just use `Web content filter proxy with multiple server modes`.

File: srcpkgs/e2guardian/template [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+maintainer="Philipp Nassua <philipp@nassua.cc>"
+license="GPL-2.0-or-later"
```
Comment (congdanhqx) [inline srcpkgs/e2guardian/template]: This project is released under GPLv2 only (without the later part), no?

**Comment by ericonr** [comment]
5.3.4 seems to be released. Remember to ping maintainers once changes have been made.

**Review by Piraty — CHANGES_REQUESTED** [review]

**Comment by abenson** [comment]
Seemingly abandoned.
